### PR TITLE
limit number of parks displayed

### PR DESCRIPTION
**Pull request by arbalisi12**
# Description

This PR includes the following proposed change(s):

- BC-215 : On the Public Advisory Dashboard, display the first three parks and a count of associated parks
- BC-216 : On the Public Advisory Dashboard, display the associated regions

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Does the change impact or break the Docker build?

- [ ] Yes
- [x] No

If Yes: Has Docker been updated accordingly?

- [ ] Yes
- [ ] No

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/admin/src/components/page/advisoryDashboard/AdvisoryDashboard.js [hunk 1]
```diff
@@ -50,9 +50,32 @@ export default function AdvisoryDashboard({ page: { setError } }) {
...
+      ),
+    ]);
+
+    const managementAreas = response[0].data;
+    const publicAdvisories = response[1].data;
+
+    const regionParksCount = managementAreas.reduce((region, item) => {
+      region[item.region.id] = (region[item.region.id] || 0) + 1;
+      return region;
+    }, {});
```
Comment (siva-ka) [inline src/admin/src/components/page/advisoryDashboard/AdvisoryDashboard.js:66]: shouldn't we have the same logic for "sections" as well?

**Review by siva-ka — APPROVED** [review]
please find my comments, thanks Arnel

File: src/admin/src/components/page/advisoryDashboard/AdvisoryDashboard.js (hunk 1 above)
Comment (arbalisi12) [reply src/admin/src/components/page/advisoryDashboard/AdvisoryDashboard.js:66]: thanks for the feedback... i'll add the sections

Comment (arbalisi12) [reply src/admin/src/components/page/advisoryDashboard/AdvisoryDashboard.js:66]: we are not adding sections for now.
sites will be suffixed to the protected-area name when the information becomes available.

**Pull request merged**